index.html

```diff
@@ -4,6 +4,61 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="css/index.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@3.3.7/dist/css/bootstrap.min.css"
+      integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Piazzolla:ital,wght@0,400;1,100&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body></body>
+  <body>
+    <div class="responsive">
+      <h1 class="text-capitalize text-center pb-5">Awesome books</h1>
+      <table class="table table-striped table-responsive table-hover">
+        <thead class="h4">
+          <tr>
+            <th scope="col">ID</th>
+            <th scope="col">TITLE</th>
+            <th scope="col">AUTHOR</th>
+            <th scope="col">DELETE</th>
+          </tr>
+        </thead>
+        <tbody id="library-list"></tbody>
+      </table>
+      <div></div>
+      <form id="form">
+        <div class="form-group">
+          <label for="title" class="sr-only">Title</label>
+          <input
+            type="text"
+            name="Title"
+            placeholder="Title"
+            id="title"
+            required
+            class="form-control book-input"
+          />
+        </div>
+
+        <div class="form-group">
+          <label for="author" class="sr-only"></label>
+          <input
+            type="text"
+            name="Author"
```

Comment:
Is there any valid reason as to why a capitalized text is being used here? not that is entirely a bad practice but it's always best to keep things simple